config/muxcable.py

```diff
@@ -188,3 +188,69 @@ def mode(state, port, json_output):
                 click.echo(tabulate(data, headers=headers))
 
         sys.exit(CONFIG_SUCCESSFUL)
+
+@muxcable.group(cls=clicommon.AbbreviationGroup)
+def prbs():
+    """Enable prbs on a port"""
+    pass
+
+@prbs.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+@click.argument('mode_value', required=True, default=None, type=click.INT)
+@click.argument('lane_map', required=True, default=None, type=click.INT)
+def enable(port, target, mode_value, lane_map):
+
```

Comment:
Suggest to add a docstring or `help=` parameter here to add a message in help dialogs (e.g., Suggest `"""Enable PRBS mode on a port"""`).

Reply:
fixed